SPECS/python-hatchling/python-hatchling.spec

```diff
@@ -75,8 +75,8 @@ install -t '%{buildroot}%{_mandir}/man1' -D -p -m 0644 \
     '%{SOURCE500}'
 
 
-%check
 %if 0%{?with_check}
```

Comment:
Same ask here: let's remove `%global with_check 0` from this spec and `%if 0%{?with_check}` from this line.